pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -51,68 +35,43 @@ name = "charlie"
 validator = true
 ```
 
-> Interesting Fact: We have three validator nodes so that we can use Polkadot's Warp Sync to quickly sync with the network. The minimum requirement for Warp Sync are three validator nodes.
```

Comment:
Again, why remove it?

Reply:
You feel it is necessary? Someone wants to deploy a parachain, there is already a lot of info.